odl/operator/operator.py

```diff
@@ -334,19 +563,21 @@ def __call__(self, x, out=None, *args, **kwargs):
                                 ''.format(out, self.range, self))
 
             if self.is_functional:
-                raise TypeError('`out` parameter cannot be used'
+                raise TypeError('`out` parameter cannot be used '
```

Comment:
I'll leave this as it is since I'm not excluding optional `out` altogether, see [here](https://github.com/odlgroup/odl/pull/116#discussion-diff-46757923R453). We need to catch this here, in `FunctionSetVector.__call__` a new logic is implemented, so no need to do anything special there.